components/Areas/Areas.js

```diff
@@ -7,23 +7,33 @@ export default function Areas() {
   return (
     <>
       <StyledPageContainer>
-        <StyledTopic>I need Information about...</StyledTopic>
-        {areas.map((area) => (
-          <StyledLink key={area.id} href={`sectors/${area.name}`}>
-            <StyledCard>
-              <StyledName>{area.name}</StyledName>
-              <StyledRoutes>Routes: {area.routes}</StyledRoutes>
-              <StyledStone>{area.stone}</StyledStone>
-              <StyledCountry>{area.country}</StyledCountry>
-            </StyledCard>
-          </StyledLink>
-        ))}
+        <StyledUl>
+          <StyledTopic>Boulderareas</StyledTopic>
+          {areas.map((area) => (
+            <StyledList key={area.id}>
+              <StyledLink href={`sectors/${area.name}`}>
+                <StyledCard>
+                  <StyledName>{area.name}</StyledName>
+                  <StyledRoutes>Routes: {area.routes}</StyledRoutes>
+                  <StyledStone>{area.stone}</StyledStone>
+                  <StyledCountry>{area.country}</StyledCountry>
+                </StyledCard>
+              </StyledLink>
+            </StyledList>
+          ))}
+        </StyledUl>
       </StyledPageContainer>
       <SearchBar />
     </>
   );
 }
 
+const StyledUl = styled.ul`
+  padding-inline-start: 0px;
+  //border: 1px green solid;
+  margin-top: 190px;
```

Comment:
I think this is not enough because for the iPhone SE preview, the "Boulderareas" is overlapping the header... I can't see your header component right now but I think using a header component would not let the content that comes afterward overlap.